### DISCUSSION TITLE
有无可能在nb-cli中加入“执行插件中脚本”的功能

### DISCUSSION THREAD

**Discussion by felinae98** [nonebot/discussions#143] [通用]
功能类似于flask cli提供的功能https://flask.palletsprojects.com/en/2.1.x/cli/#plugins
目前观察到社区中的一些插件通过别的方式实现了类似的功能，例如https://github.com/SK-415/HarukaBot/blob/master/src/plugins/haruka_bot/cli/bot.py
个人感觉也需要这个功能，所以未来是否有机会在nb-cli中实现这个功能

**Comment by mnixry** [comment]
感谢提议

目前，NoneBot CLI 将作为一项[开源之夏暑期2022](https://nonebot.dev/ospp-2022.html#nonebot2-命令行-cli-交互体验升级)的项目
会在今年暑期迎来一次较大规模的重构，我们已经正在考虑将您的提议加入本次项目的Roadmap
我们将在近期在NoneBot CLI仓库创建一个Issue以对项目Roadmap的完成情况进行跟踪

如果您想加入对NoneBot的改进工作，您也可以[参加报名](https://nonebot.dev/ospp-2022.html)成为学员来协力我们进行改进工作
